scm-ui/ui-webapp/src/users/components/UserForm.tsx

```diff
@@ -178,24 +204,41 @@ class UserForm extends React.Component<Props, State> {
                 helpText={t("help.mailHelpText")}
               />
             </div>
-          </div>
-          {passwordChangeField}
-          <div className="columns">
-            <div className="column">
+            <div className="column is-full">
               <Checkbox
-                label={t("user.active")}
-                onChange={this.handleActiveChange}
-                checked={user ? user.active : false}
-                helpText={t("help.activeHelpText")}
+                label={t("user.externalFlag")}
+                onChange={this.handleExternalFlagChange}
+                checked={!!user?.external && user.external}
```

Comment:
I think an undefined check is more readable than a double negation.

Reply:
I simplified the check